### PR TITLE
BCP-update-when-billed: INFODEV-826

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1901553aee/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-46cfb46b8f/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson For your review.

**Comment by cyrichardson** [comment]
On it, @RackerKate.

File: content/general/when-am-i-billed-for-my-account.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2018-02-22'
+last_modified_by: Kate Dougherty
 product: undefined
 product_url: undefined
 ---
 
 When you sign up for products under the Managed Infrastructure or Managed
-Operations SysOps service level, your credit card is charged a US $1.00 authorization charge to confirm that it is an active card. The authorization charge returns to your card normally within 24-72 hours.
+Operations SysOps service level, your credit card is charged a US $1.00 authorization charge to confirm that it is an active card. The amount of the authorization charge is refunded to your card within 24-72 hours.
```
Comment (cyrichardson) [inline content/general/when-am-i-billed-for-my-account.md]: within 24 to 72 hours. 

(I had to look this one up :-) - https://rackerlabs.github.io/docs-rackspace/style-guide/style/punctuation.html?highlight=hyphens#dashes - last sentence in the section.

**Comment by cyrichardson** [comment]
@RackerKate This one was so short that I put my only review comment inline. I hope that's OK. Please let me know if you have any questions. Thanks!

**Comment by RackerKate** [comment]
@cyrichardson That's fine, thanks.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-8991e1ad1a/how-to/)

**Pull request merged**